docs/manual/developer/04_style_guide.md

```diff
@@ -348,8 +348,9 @@ Value must be low, medium, or high.
 * Shall use fully-qualified collection names [(FQCN)](https://ansible-lint.readthedocs.io/rules/fqcn/). e.g. use `ansible.builtin.lineinfile:` instead of only `lineinfile:`
 * Shall use specific Ansible modules whenever possible instead of just calling system commands with `command`, `shell` or `raw` modules
     * When no specific Ansible module is available, `command` module should be preferred instead of `shell` or `raw` modules
+* Shall define short and objective task names that reflect the end state of a machine
+* Task names shall be prefixed by `{{{ rule_title }}}`, e.g. `- name: "{{{ rule_title }}} - Ensure Correct Banner"`
```

Comment:
Do we want to note that Title case shall be used here? That is part of the Ansible Lint rules IIRC.

Reply:
I though about this when updating the task name in the example, but forgot to include. :) I think it makes sense to make explicit here even if is part of Ansible Lint. Thanks for noticing. I will include this rule.

Reply:
Done.